### ISSUE TITLE
Broken OBO Foundry ontology URL on Dashboard Report pages

### ISSUE THREAD

**Issue by jseager7** [OBOFoundry/OBO-Dashboard#13]
The Dashboard Report pages have a link to the OBO Foundry page for the ontology, but it looks like Jinja isn't converting something to a string properly:
```
http://obofoundry.org/ontology/<class 'jinja2.utils.Namespace'>.html
```
This should be correctly rendered as (using PHIPO as an example):
```
http://obofoundry.org/ontology/phipo.html
```

**Comment by beckyjackson** [comment]
Thanks for the catch! I'm making a fix for this now.

**Comment by beckyjackson** [comment]
Fixed by 3f07437
